### PR TITLE
abl export button float fix TM-3198

### PR DESCRIPTION
**Pull request by SidelineCory24**
To-do

- [x] export button should not float when Statistics drop down is open (AO & CDO view)

**Note: Statistics do not show for Bureau or Post users**

### CODE REVIEW

File: src/sass/_availableBidders.scss [hunk 1]
```diff
@@ -211,9 +211,7 @@
   margin-bottom: -50px;
   .export-button-container {
     button {
-      position: absolute;
-      right: 50px;
-      top: 100px;
+      position: sticky;
```
Comment (elizabeth-jimenez) [inline src/sass/_availableBidders.scss:214]: We might not need any styling on the button, looks like?

Comment (SidelineCory24) [reply src/sass/_availableBidders.scss]: done

File: src/sass/_availableBidders.scss [hunk 2]
```diff
@@ -209,12 +209,8 @@
...
   align-items: baseline;
   margin-bottom: -50px;
-  .export-button-container {
-    button {
-      position: absolute;
-      right: 50px;
-      top: 100px;
-    }
+  button {
+    margin-right: 0;
```
Comment (elizabeth-jimenez) [inline src/sass/_availableBidders.scss:213]: this makes the button flat with the table

**Review by elizabeth-jimenez — APPROVED** [review]
🐱

**Pull request merged**